RELEASENOTES.md

```diff
@@ -0,0 +1,35 @@
+# HBASE  hbase-filesystem-1.0.0-alpha1 Release Notes
```

Comment:
this file needs a ASF license header. if you run release doc maker with the `--license` option it should do it for you.